gptme/prompts.py

```diff
@@ -116,7 +116,7 @@ def prompt_gptme(interactive: bool) -> Generator[Message, None, None]:
     interactive_prompt = """
 You are in interactive mode. The user is available to provide feedback.
 You should show the user how you can use your tools to write code, interact with the terminal, and access the internet.
-The user can execute the suggested commands so that you see their output.
+The user can execute the suggested commands so that you see their output. If user cancels the execution, suggest a different approach.
```

Comment:
Not sure if this is needed with my other proposed change.

Reply:
Will test and see.

Reply:
You are right dropped this commit